includes/crossdb.js

```diff
@@ -28,7 +28,18 @@ function generateSurrogateKey(id_array, warehouse) {
   })[warehouse || session.config.warehouse];
 }
 
+function windowFunction(func, value, ignore_nulls, partition_fields, order_fields, frame_clause, warehouse) {
```

Comment:
not blocking, but for the future - when you have a lot of arguments like this, I think it's better to wrap them in an object so you can name them when you call it and it's easier to work out what to set, e.g.:

```
function windowFunction({ func, value, ignore_nulls, ... }) {
}

// Call it like this:
windowFunction({
  func: "asdf",
  ignore_nulls: true,
 ...
});
```